testng-curl/src/test/java/com/browserstack/BStackDemoTest.java

```diff
@@ -0,0 +1,185 @@
+package com.browserstack;
+
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import org.testng.Reporter;
+import org.testng.annotations.Test;
+
+public class BStackDemoTest extends SeleniumTest {
+    @Test
+    public void visitBStackDemo() throws Exception {
+    	
+
+        // navigate to bstackdemo
+        driver.get("https://www.bstackdemo.com");
+
+        
+        // Hi
+        // Check the title
+        Assert.assertTrue(driver.getTitle().matches("StackDemo"));
+
+        // Save the text of the product for later verify
+        String productOnScreenText = driver.findElement(By.xpath("//*[@id=\"1\"]/p")).getText();
+        // Click on add to cart button
+        driver.findElement(By.xpath("//*[@id=\"1\"]/div[4]")).click();
```

Comment:
Test case is - visitBStackDemo
So test should terminate at Line 19, post that the steps are to add product to cart which is a redundant step else change the test name